isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -41,36 +42,337 @@
       Optional output files can be selected to provide more information for analyzing the
       results.  <b>BUNDLEOUT_TXT</b> provides an overall summary of the bundle adjustment.
       It lists the user input parameters selected and tables of statistics for both the
-      images and the points.   The image statistics can also be written to a separate
+      images and the points. The image statistics can also be written to a separate
       <b>CSV</b> file and likewise for the point statistics with the <b>OUTPUT_CSV</b>
       option selected.  <b>RESIDUALS_CSV</b> provides a table of the measured image
       coordinates and the final <def>sample</def>, <def>line</def>, and overall residuals
       in both millimeters and pixels.
     </p>
 
+    <h3>Observation Equations</h3>
+
+    <p>
+      The <b>jigsaw</b> application attempts to minimize the reprojective error within the control network.
+      For each control measure and its control point in the control network, the reprojective error is
+      the difference between the control measures image coordinate and the image coordinate that the control
```

Comment:
Suggested: "the difference between the control measures image coordinates **(typically determined through image registration)**" Then link to image registration [documentation](https://github.com/USGS-Astrogeology/ISIS3/wiki/Automatic_Registration)